specification/web/resource-manager/Microsoft.Web/2016-03-01/Recommendations.json

```diff
@@ -323,8 +329,10 @@
           "type": "string"
         },
         "recommendationId": {
+          "format": "uuid",
           "description": "A GUID value that each recommendation object is associated with.",
-          "type": "string"
+          "type": "string",
```

Comment:
SDK breaking change.

Reply:
Client validation for (uuid, byte ) will cause SDK breaks, but is acceptable and actually a good change.